DEPLOY.md

```diff
@@ -19,6 +19,7 @@ The following configuration values must be set via environment variables.
 
 | Variable Name                | Description                                                                                                                                                                        |
 | ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `HOSTNAME`                   | The hostname of the server (e.g., `https://mycustomdomain.senecacollege.ca`)                                                                                                       |
```

Comment:
host.docker.internal now, not sure if we should add to description

Reply:
Good idea, updated docs to be more clear.